cmd/ping/main.go

```diff
@@ -91,7 +91,9 @@ func serveHTTP(ctlConf config, h healthcheck.Handler) func() {
 
 	go func() {
 		logger.Info("starting HTTP Server...")
-		logger.WithError(srv.ListenAndServe()).Fatal("could not start HTTP server")
+		if err := srv.ListenAndServe(); err != http.ErrServerClosed {
+			logger.WithError(err).Fatal("could not start HTTP server")
```

Comment:
```suggestion
			logger.WithError(err).Fatal("Could not start HTTP server")
```

We are super inconsistent with this 😬 , but log statements should be start with acapitall letter. I know you are copy pasting from the previous log message, but I'd like to take the opportunity to fix this while you are in there 😄 

Otherwise, this is great. Happy to merge with this small nit change. Thanks!